stable/cluster-autoscaler/Chart.yaml

```diff
@@ -2,8 +2,8 @@ apiVersion: v1
 description: Scales worker nodes within autoscaling groups.
 icon: https://github.com/kubernetes/kubernetes/blob/master/logo/logo.png
 name: cluster-autoscaler
-version: 3.2.0
-appVersion: 1.13.1
+version: 3.3.0
```

Comment:
Thanks! Will you please rebase and bump this to `3.4.0`?

Reply:
👍 Rebased.